frontend/src/Pages/AdminDashboard.tsx

```diff
@@ -22,8 +22,10 @@ export default function AdminDashboard() {
     const totalActivity = convertSeconds(activityData.total_weekly_activity);
 
     return (
-        <div className="px-8 py-4">
-            <h1 className="text-5xl">{activityData.facility_name}</h1>
+        <div className="flex flex-col space-y-6 overflow-x-auto rounded-lg p-4">
+            <div className="pt-8">
+                <h1>{activityData.facility_name}</h1>
+            </div>
```

Comment:
This is actually the only one that should stay as-is! The facility name is larger on the designs so i think we should just maybe keep it that way?